### PR TITLE
fix(ui-components) - align lint:fix script with others in monorepo

### PR DESCRIPTION
**Pull request by devinea**
fix(ui-components) - align lint:fix script with others in monorepo

Notice issues on:
https://github.com/SAP/open-ux-tools/pull/782
https://github.com/SAP/open-ux-tools/pull/812

### CODE REVIEW

**Comment by tobiasqueck** [comment]
>  some lint:fix scripts behave differently, resulting in missing lint errors

Do you have an example, and shouldn't we fix those?

**Comment by tobiasqueck** [comment]
> I think it is safer to use lint script with --fix parameter

I disagree. When you work, you use the scripts within the module, and I expect that when we run a script for the whole repo that it does for each module what the script in the module does.

**Review by tom-sap — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
         "build": "nx run-many --target=build --all",
         "format": "pnpm recursive run format",
         "lint": "nx run-many --target=lint --all",
-        "lint:fix": "nx run-many --target=lint:fix --all",
+        "lint:fix": "nx run-many --target=lint --fix --all",
```
Comment (tobiasqueck) [inline package.json:39]: Please consider removing this change again, and make sure that all modules work as expected, to keep running something within a module consistent with running it from root.

Comment (devinea) [reply package.json]: Done in https://github.com/SAP/open-ux-tools/pull/813/commits/394d782198507cc923adec228708c1cc05244333

**Review by tobiasqueck — DISMISSED** [review]

**Review by tobiasqueck — APPROVED** [review]
Is the formatting change of the `Icons.tsx` on purpose? If yes, then all is good.

**Review by unseen1980 — APPROVED** [review]

**Comment by devinea** [comment]
Thanks all

**Pull request merged**